docs/make_projection.py

```diff
@@ -162,7 +165,14 @@ def prj_class_sorter(cls):
             # Get instance arguments and number of plots
             instance_args = SPECIFIC_PROJECTION_KWARGS.get(prj, {})
 
-            prj_inst, instance_repr = create_instance(prj, instance_args)
+            try:
```

Comment:
Note that I added this here because I don't have PROJ 9.6 installed locally, so this didn't work when instantiated. Maybe there is a better way to approach this?